### PR TITLE
Search Bar Update

### PR DESCRIPTION
**Pull request by drewjhart**
Addressing Issue #183: "Hide Search Bar"

Update to ensure search bar is hidden when not on Explore page.

### CODE REVIEW

File: web/src/components/Nav.jsx [hunk 1]
```diff
@@ -34,22 +33,9 @@ export default function PrimarySearchAppBar({ setSearchInput, searchInput }) {
               </Typography>
             </NavLink>
             <img src={ComingSoon} alt="Coming Soon!!" style={{height: 50, marginLeft: 50, marginRight: 20}} />
+            {matchSearchBar.isExact ? <SearchBar setSearchInput={setSearchInput} searchInput={searchInput} /> : null}
+            {!matchGamesPage.isExact ? console.log("found it") : null}
```
Comment (maniramezan) [inline web/src/components/Nav.jsx:37]: Is this a debug code that's left here?

**Review by maniramezan — CHANGES_REQUESTED** [review]

File: web/src/components/Nav.jsx (hunk 1 above)
Comment (drewjhart) [reply web/src/components/Nav.jsx]: Thanks for catching that - that's old code. I've pushed a new commit that matches what I showed you on slack

**Review by maniramezan — APPROVED** [review]
LGTM, nice job ❤️

**Review by zach2025 — APPROVED** [review]
Looks good to me too!

**Pull request merged**